packages/amplication-data-service-generator/src/server/create-dotenv.template.env

```diff
@@ -5,4 +5,5 @@ POSTGRESQL_URL=postgres://${dbUser}:${dbPassword}@${dbHost}:${dbPort}${dbName}
 BCRYPT_SALT=10
 COMPOSE_PROJECT_NAME=amp_${appId}
 JWT_SECRET_KEY=Change_ME!!!
-JWT_EXPIRATION=2d # more options in https://github.com/nestjs/jwt#jwtservicedecodetoken-string-options-decodeoptions-object--string
\ No newline at end of file
+# more options in https://github.com/nestjs/jwt#jwtservicedecodetoken-string-options-decodeoptions-object--string
```

Comment:
remove this link, all the details and instructions should be included in our docs site including links to external docs